ParticleSpy/segptcls.py

```diff
@@ -74,7 +75,8 @@ def threshold(data, process_param):
     if process_param["threshold"] == "li":
         thresh = threshold_li(data)
     if process_param["threshold"] == "local":
-        thresh = threshold_local(data,process_param["local_size"])
+        w_size = 2*process_param["local_size"]+1
```

Comment:
I understand this ensures an odd number but it makes setting the size for local thresholding a bit more obscure. i.e. people need to know to use a value half of what they really want. Would it be better to check if local_size is even and if so change to an odd number?

e.g. if process_param['local_size'] % 2 == 0:
          process_param['local_size'] += 1

What do you think?